test/references/character_escaping/problematic_characters.docx.txt

```diff
@@ -209,7 +209,7 @@
             </w:pPr>
             <w:r>
               <w:rPr/>
-              <w:t xml:space="preserve">&amp; % $ # _ { } ~ ^ \ &lt; &gt; " ' </w:t>
```

Comment:
Not familiar enough with the docx output to know why this has changed.

Reply:
the inner string ended with a space before and to keep that in the visual output, WriteDocx sets the space preserve attributes for such text tags. Now it ends with a backtick so the attribute disappears again. I don't think it ended with space on purpose before, so that doesn't matter